wallet/lib/pages/purchase_item/purchase_item_screen.dart

```diff
@@ -126,7 +131,7 @@ class _PurchaseItemContentState extends State<PurchaseItemContent> {
 
     return NftImageWidget(
       url: thumbnailUrl,
-      opacity: 0.4,
+      opacity: viewModel.isViewingFullNft ? 0.0 : 0.4,
```

Comment:
What you mean by isViewingFullNFT? Can you also attach a demo video please.

Reply:
https://user-images.githubusercontent.com/106388520/190639541-f2c43480-11f6-41ee-8fa4-b514ac60d71e.mp4

Reply:
> What you mean by isViewingFullNFT? Can you also attach a demo video please.

We have declared a variable which keeps track of long Press, which means user wants to view full NFT